ui/components/KubeStatusIndicator.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import styled from "styled-components";
+import { Condition } from "../lib/api/app/source.pb";
+import { computeMessage, computeReady } from "../lib/utils";
+import Flex from "./Flex";
+import Icon, { IconType } from "./Icon";
+
+type Props = {
+  className?: string;
+  conditions: Condition[];
+};
+
+function KubeStatusIndicator({ className, conditions }: Props) {
+  const ready = computeReady(conditions) === "True";
+  const readyText = ready ? "Ready" : computeMessage(conditions);
+  const color = ready ? "success" : "alert";
+
+  return (
+    <Flex start className={className} align>
+      <Icon color={color} size="base" type={IconType.Circle} text={readyText} />
+    </Flex>
+  );
+}
+
+export default styled(KubeStatusIndicator).attrs({
+  className: KubeStatusIndicator.name,
+})``;
```

Comment:
💕 ❤️ 💕

Reply:
> @sympatheticmoose are all sortable fields intended to have a sort arrow? Right now only the field that is actively being sorted has one

I believe that is the design... @bia could you confirm? I am assuming the column head is clickable and thats what applies/flips the sort?

Reply:
Yes, in the current design only the field that is actively being sorted has an arrow (facing up or down based on sort direction). The other column headers that can sort the table should be clickable and the sort would be applied on click.